docs/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+
+[compat]
+Documenter = "~0.25"
```

Comment:
```suggestion
Documenter = "~0.26"
```

Reply:
Will this cause problems? Other repos use 0.25. Maybe we can upgrade them all in a second wave.

Reply:
It won't cause problems.